File: fixtures/RH002/before/CalculatorSpec.hs
module CalculatorSpec (spec) where

import Test.Hspec
import Data.Maybe (isJust)
import Calculator (safeDivide)

spec :: Spec
spec = do
  describe "safeDivide" $ do
    it "divides two numbers" $ do
      let result = safeDivide 6 3
      result `shouldBe` Just 2

File: fixtures/RH003/negative/before/CalculatorSpec.scala
import org.scalatest.flatspec.AnyFlatSpec

class CalculatorSpec extends AnyFlatSpec {
  "Calculator" should "add two numbers" in {
    val ignore = false
    if (!ignore) {
      assert(Calculator.add(1, 2) == 3)
    }
  }
}
